src/api/config/repos/repos.go

```diff
@@ -69,3 +63,7 @@ func loadReposFromFile(filePath string) (Repos, error) {
 	}
 	return yamlStruct.Repos, nil
 }
+
+func strToPtr(s string) *string {
```

Comment:
It is in the handlers package, which imports repos so it can't be imported here. I'll move this to a new util package that only holds basic functions like this.